### PR TITLE
Add filestore instance support to GCP janitor

### PR DESCRIPTION
**Pull request by chemikadze**

### CODE REVIEW

File: cmd/janitor/gcp_janitor.py [hunk 1]
```diff
@@ -283,7 +288,9 @@ def collect(project, zones, age, resource, filt, clear_all):
         'list',
         '--format=json(name,creationTimestamp.date(tz=UTC),createTime.date(tz=UTC),zone,region,isManaged)',
         '--project=%s' % project])
-    if resource.condition == 'zone' and resource.name != 'sole-tenancy' and resource.name != 'network-endpoint-groups':
+    if (resource.condition == 'zone'
```
Comment (chizhg) [inline cmd/janitor/gcp_janitor.py:291]: What's the reason for deleting `resource.name != 'network-endpoint-groups'`? Was it by mistake?

File: cmd/janitor/gcp_janitor.py [hunk 2]
```diff
@@ -318,6 +325,10 @@ def collect(project, zones, age, resource, filt, clear_all):
                 # subcommands want just the name, not the full URL, so strip it.
                 colname = item[resource.condition].rsplit('/', 1)[-1]
                 log('looking for items in %s=%s' % (resource.condition, colname))
+            elif resource.group == 'filestore':
```
Comment (chizhg) [inline cmd/janitor/gcp_janitor.py:328]: I'm not familiar with `filestore` so not entirely sure how it works. Were you able to run this script against your own project to make sure this does not break?

File: cmd/janitor/gcp_janitor.py (hunk 1 above)
Comment (chemikadze) [reply cmd/janitor/gcp_janitor.py:291]: Great catch, thanks! Wanted to move per-resource match to separate lines, lost during reformatting.

File: cmd/janitor/gcp_janitor.py (hunk 2 above)
Comment (chemikadze) [reply cmd/janitor/gcp_janitor.py:329]: Yes, I have verified it with my project having filestore instances.

**Comment by chemikadze** [comment]
@stevekuznetsov @alvaroaleman do you mind taking a look at the PR? Thanks in advance!

**Review by alvaroaleman — APPROVED** [review]

**Pull request merged**